app/views/static_pages/team.html.erb

```diff
@@ -326,19 +326,14 @@
                 <%= link_to image_tag("partners/CODE.png", class: "logo"),
                 "http://codedocumentary.com/", target: "_blank" %>
             </div>
-             <div class="partner col-sm-4">
+            <div class="partner col-sm-4">
                 <%= link_to image_tag("partners/JETBRAINS.png", class: "logo"),
                 "https://www.jetbrains.com/", target: "_blank" %>
             </div>
             <div class="partner col-sm-4">
                 <%= link_to image_tag("partners/devbootcamp.png", class: "logo"),
                 "https://devbootcamp.com/", target: "_blank" %>
             </div>
-         
-          </div><!-- /row -->
-          <div class="row">
-       
-           
         </div><!-- /row -->
```

Comment:
Thank you for explaining! I appreciate it. Learned something new :)  Let me know if I did it right this time.